src/resources/2022/GreyGoose.ts

```diff
@@ -0,0 +1,53 @@
+import { have as have_ } from "../../lib";
+import { get as getModifier } from "../../modifier";
+import { get } from "../../property";
+import { $familiar, $item, $skill } from "../../template-string";
+
+export const goose = $familiar`Grey Goose`;
+
+export function have(): boolean {
+  return have_(goose);
+}
+
+export function currentExperience(): number {
+  return goose.experience || have_($familiar`Shorter-Order Cook`)
+    ? 81 + (have_($item`blue plate`) ? 19 : 0)
+    : 0;
+}
+
+export function currentWeight(): number {
+  return Math.min(Math.floor(Math.sqrt(currentExperience())), 20);
+}
+
+export function expectedDrones(weight = currentWeight()): number {
+  return Math.max(0, weight - 5);
+}
+
+export function expectedExperience(weight = currentWeight()): number {
```

Comment:
this needs a grey you switch. if the user is in grey you, the experience skills grant you (weight - 5)^2 fullstats rather than the cube of weight-5 substats. this one probably needs a docstring due to this small complexity.